mtags/src/main/scala-2/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -407,23 +405,29 @@ abstract class PcCollector[T](
         case imp: Import if filter(imp) =>
           val res = imp.selectors.foldLeft(traverse(acc, imp.expr)) {
             case (acc, sel)
-                if soughtFilter(sought => sought.exists(_.name == sel.name)) =>
+                if soughtFilter(_.decodedName == sel.name.decoded) =>
               val positions =
                 if (sel.rename != null && !sel.rename.isEmpty)
                   Set(
                     sel.renamePosition(pos.source),
                     sel.namePosition(pos.source)
                   )
                 else Set(sel.namePosition(pos.source))
-              val symbol = imp.expr.symbol.info.member(sel.name)
+              val symbol = imp.expr.symbol.info.member(sel.name) match {
+                case ns: NoSymbol =>
```

Comment:
Do you have any ideas in what cases we get `NoSymbol` here?
It would be helpful to note in a comment why for future us 👍

Reply:
So, this change is the fix for `import foo.bar.@@` right?

Reply:
We can get `NoSymbol` when we import `_`, `*`, `given`, or when the names don't match (here "@@" doesn't match "$at$at"). Then we try to find member based on decodedName

Reply:
I see, thanks!
I think this comment
> We can get NoSymbol when we import _, *, given, or when the names don't match (here "@@" doesn't match "$at$at"). Then we try to find member based on decodedName

is well describing why we do this, and it would be good to comment :+1: